addons/hr_timesheet/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,74 @@
+---Fields in module 'hr_timesheet'---
+hr_timesheet / account.analytic.line    / employee_id (many2one)        : NEW relation: hr.employee
```

Comment:
Fill this with the SQL equivalent to:
`self.env['hr.employee'].search([('user_id', '=', result['user_id'])], limit=1).id`

Reply:
Yes, this must be filled with this.